configure.ac

```diff
@@ -166,13 +166,13 @@ else
     AC_MSG_NOTICE(Developer mode disabled)
 fi
 
-if test ! -f $srcdir/src/version.c; then
+if test ! -f $srcdir/src/version.c -a ! -f src/version.c ; then
     AC_MSG_WARN(unable to determine ccache version)
     echo "const char CCACHE_VERSION@<:@@:>@ = \"unknown\";" >src/version.c
```

Comment:
```
% mkdir build
% cd build
% ../configure
[...]
configure: Developer mode enabled
../configure: line 6214: src/version.c: No such file or directory
configure: WARNING: unable to determine ccache version
../configure: line 6223: src/version.c: No such file or directory
```

So move `mkdir -p .deps src unittest` further up in the file?

Reply:
Wonder why the distcheck didn't catch this - oh wait, it was developer mode only